src/main/java/frc/robot/utilities/VictorConfiguration.java

```diff
@@ -0,0 +1,104 @@
+package frc.robot.utilities;
+
+import com.ctre.phoenix.motorcontrol.*;
+import com.ctre.phoenix.motorcontrol.can.VictorSPXConfiguration;
+import com.ctre.phoenix.motorcontrol.can.VictorSPXPIDSetConfiguration;
+
+public class VictorConfiguration {
+
+    /**
+     * This class is meant to be used as an effective way to give several VictorSPXs with the same
+     * wanted configurations the same settings without a need to change each one separately
+     * This uses the CTRE/WPILib VictorSPXConfiguration class and adds some likely needed parameters
```

Comment:
```suggestion
     * Wanted configurations in the same settings without a need to change each one separately.
     * This uses the CTRE/WPILib VictorSPXConfiguration class and adds some likely needed parameters.
```
Rephrase "wanted configurations in the same settings without a need to change each one separately".